include/deal.II/grid/reference_cell.h

```diff
@@ -564,15 +564,32 @@ class ReferenceCell
 
   /**
    * Determine the orientation of the current entity described by its
-   * vertices @p var_1 relative to an entity described by @p var_0.
+   * vertices @p vertices_1 relative to an entity described by @p var_0.
```

Comment:
what is var_0, vertices_0?

Reply:
Oh, I fixed one but not the other :-( I'll fix that up in a follow-up patch -- if you don't mind, allow me to just roll that into some random commit, I've already made the edit in my local copy.

Reply:
You might have noticed that previously, *both* of the `@p` references were to function arguments that weren't actually named that way :-)